examples/09. Advanced Attributes.md

```diff
@@ -47,7 +47,7 @@ Retrieves the coupon with the given ID.
 Returns a list of your coupons.
 
 + Parameters
-    + limit = `10` (optional, number)
+    + limit: 10 (optional, number)
```

Comment:
Most of the other examples you've removed `optional` as it's optional by default. Is there any reason you didn't for this particular case?